modules/azure_aks/tf_module/variables.tf

```diff
@@ -5,9 +5,9 @@ data "azurerm_subscription" "current" {}
 data "azurerm_client_config" "current" {}
 
 data "azurerm_subnet" "opta" {
-  name                 = "opta-${var.env_name}-subnet"
+  name                 = var.private_subnet_name
```

Comment:
Have we confirmed pre-existing subnet names are not impacted via a test on a pre-existing cluster?

Reply:
yup! I tried it out with my pre-existing azure test cluster